cmd/root.go

```diff
@@ -178,23 +180,37 @@ func Execute() {
 	)
 
 	if err := c.cmd.Execute(); err != nil {
+		exitCode := -1
+		var ecerr errext.HasExitCode
+		if errors.As(err, &ecerr) {
+			exitCode = int(ecerr.ExitCode())
+		}
+
+		hint := ""
+		var herr errext.HasHint
+		if errors.As(err, &herr) {
+			hint = herr.Hint()
+		}
+
```

Comment:
I think this can be move down to the place we actually set it to `fields`

Reply:
Resolved by https://github.com/k6io/k6/pull/2046/commits/bbd254196f60e752f37b8d9a2eedd70f9f617e20

Reply:
actually, wait, this can be simplified more

Reply:
yeah I meant ... replacing the `if hint != '"` :) ... but probably should've worded it better

Reply:
Resolved by https://github.com/k6io/k6/pull/2046/commits/a7b85eba6b495e370f699ef855adada36bdb7eb3